src/main/java/trade/wayruha/mexc/constant/ApiEndpoints.java

```diff
@@ -2,6 +2,14 @@
 
 public class ApiEndpoints {
 
+    //Market Data Endpoints
+    /** If the symbol is not sent, all symbols will be returned in an array.*/
+    public static final String SYMBOL_PRICE_TICKER_API_PATH = "/api/v3/ticker/price";
+
+    /** If the symbol is not sent, all symbols will be returned in an array.*/
```

Comment:
is this comment relevant?

Reply:
The seconf one not relevant, you are right.